src/Generation/Generator/Renderer/Public/ReturnType/From/StringArrayReturnType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Generator.Model;
+
+namespace Generator.Renderer.Public;
+
+internal class StringArrayReturnType : RenderableArrayReturnType<GirModel.String>
+{
+    public override RenderableReturnType Create(GirModel.ReturnType returnType)
+    {
+        // Calling Supports() first guarantees that AsT1 will succeed.
```

Comment:
If the interface is implemented here the comment should be obsolete. Please remove comment.